Add lovakush.adoc (#5862)
jenkins-infra/jenkins.io: content/_data/authors/lovakush.adoc

---
name: "Lovakush"
github: lovakush

> linkedin: lova kush

Reviewer: LinkedIn ID should not include a space.  I believe that https://www.linkedin.com/in/lova-kush-4a568b1b6/ is you.  If that is correct, then this is the change that is needed.
```suggestion
linkedin: lova-kush-4a568b1b6
```

---

> Lovakush is a 3rd year Computer Science student from Kalinga Institute of Industrial Technology, Odisha, India. His interest in Jenkins Configuration as Code (JCasC) drift detector his participation in the Jenkins GSOC 2023 program. Highly interested and contributing on open source projects.

Reviewer: Sentence per line makes it easier for reviewers.  A few phrasing changes for your consideration.  Optional, since this is your description, you have complete control of the content.  If you decide to not apply this change, please note that as a comment in the pull request so that we do not delay merging the pull request.
```suggestion
Lovakush is a 3rd year Computer Science student from Kalinga Institute of Industrial Technology, Odisha, India.
He is interested in Jenkins Configuration as Code (JCasC) drift detector as part of the Jenkins GSOC 2023 program. 
Highly interested in contributing to open source projects.
```

Reviewer 2: Hello @Lovakush ,
Thank you for your interest in GSoC and submit your bio.

Could you help me clarify wether you are submitting it as project mentor or as candidate ?  The best is to reach out to me via any convenient channel (community.jenkins.io or gitter) or better via my eMail (jean-marc@meessen-web.org). 

/- Jmm
Jenkins GSoC 2023 Org Admin

Author: I am submitting it as a candidate.
